app/assets/javascripts/discourse/components/search-advanced-options.js.es6

```diff
@@ -106,6 +106,15 @@ export default Ember.Component.extend({
         ? this.inOptionsForUsers.concat(this.inOptionsForAll)
         : this.inOptionsForAll
     });
+
+    if (this.currentUser.get("staff") && this.statusOptions.length === 5) {
```

Comment:
I needed to add `this.statusOptions.length === 5` here because it seemed that `init` got called about 3-4 times every time I reloaded the site, which resulted in 3-4 "are unlisted" entries in the menu. Any way to achieve that on a more cleaner way than this *dirty* hack?